src/mps/abstractmps.jl

```diff
@@ -1734,9 +1733,13 @@ splitblocks(::typeof(linkinds), M::AbstractMPS; tol = 0) =
 #
 
 BroadcastStyle(MPST::Type{<:AbstractMPS}) = Style{MPST}()
+BroadcastStyle(::DefaultArrayStyle{1}, ::Style{MPST}) where {MPST<:AbstractMPS} = Style{MPST}()
+BroadcastStyle(::Style{MPST}, ::DefaultArrayStyle{1}) where {MPST<:AbstractMPS} = Style{MPST}()
```

Comment:
I thought that you weren't supposed to define both orderings (i.e. https://docs.julialang.org/en/v1/manual/interfaces/#writing-binary-broadcasting-rules).

Reply:
I need to fix this anyway since it's too restrictive (should be `N` and not `1`). I'll push a fix.